### PR TITLE
Default Bombers Targeting Priority 

### PR DESCRIPTION
**Pull request by Ezahiel**
Set Bombers as Default priority for AA, followed by same priority for Fighters and the rest of Vtols, except Scouts which are on lowest priority. 
Bombers are set as follows: Bombers , Torpedo Bombers , Torpedo Gunship (armada/cortex)

### CODE REVIEW

File: luarules/gadgets/unit_targetting_priorities.lua [hunk 1]
```diff
@@ -2,7 +2,7 @@ function gadget:GetInfo()
 	return {
 		name = 'Units Targetting Prioritie',
 		desc = 'Adds a priority command/button to set the wanted target prioritizing',
-		author = 'Doo',
+		author = 'Doo', --additions wilkubyk
```
Comment (WatchTheFort) [inline luarules/gadgets/unit_targetting_priorities.lua:5]:
```suggestion
		author = 'Doo, wilkubyk',
```

File: language/en/interface.json [hunk 2]
```diff
@@ -219,8 +219,9 @@
 			"onoff_tooltip": "Active state: turn a unit on/off",
 			"Fighters": "Fighters",
 			"Bombers": "Bombers",
+			"Vtols": "Vtols",
 			"No priority": "No priority",
-			"setpriority_tooltip": "Set target priority Fighters/Bombers/no priority",
+			"setpriority_tooltip": "Set target priority Fighters/Bombers/Vtols/no priority",
```
Comment (WatchTheFort) [inline language/en/interface.json:224]: I would not update this, as we are going to remove the command and just have the priorities hardcoded. As well, "Vtols" is not meaningful to the player.

**Review by WatchTheFort — CHANGES_REQUESTED** [review]

File: language/en/interface.json (hunk 2 above)
Comment (Ezahiel) [reply language/en/interface.json:224]: @WatchTheFort its all fine since cmd been hidden , so it doesnt matter, as tooltip wont be displayed.

Comment (WatchTheFort) [reply language/en/interface.json:224]: Any time an entry in one the translation files changes, it gets flagged by Transifex for re-translation

Comment (Ezahiel) [reply language/en/interface.json:224]: Ok, will remember. Ty

**Pull request merged**